extensions/aggregation/README.md

```diff
@@ -0,0 +1,234 @@
+
+## STAC Endpoints
+
+| Endpoint     | Returns                                                        | Description |
+| ------------ | -------------------------------------------------------------- | ----------- |
+| `/aggregate` | AggregationCollection | Retrieves an aggregation of the group of Items matching the provided predicates |
+
+The `/aggregate` endpoint behaves similarly to the `/search` endpoint, but instead of returning an ItemCollection of Items, it instead returns aggregated information over the same matching Items in the form of an **AggregationCollection** of **Aggregation** entities.
+
+If the `/aggregate` endpoint is implemented, it is **required** to add a link with the `rel` type set to `aggregate` to the `links` array in root entity (`/`) that refers to the aggregate endpoint in the `href` property. This link **should** also have a field `aggregations` having a string array value that advertises the available values for the `aggregations` query parameter. 
+
+## Filter Parameters and Fields
+
+The filters for `/aggregate` are the same as those for `/search` that are semantically meaningful (e.g., limit has no meaning when doing aggregations). These filters are passed as query string parameters or JSON 
+entity fields.  For filters that represent a set of values, query parameters should use comma-separated 
+string values and JSON entity attributes should use JSON Arrays. 
+
+| Parameter    | Type             | Description |
+| -----------  | ---------------- | ----------- |
+| datetime     | string           | Single date+time, or a range ('/' seperator), formatted to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). Use double dots `..` for open date ranges. |
+| bbox         | \[number]        | Requested bounding box.  Represented using either 2D or 3D geometries. The length of the array must be 2*n where n is the number of dimensions. The array contains all axes of the southwesterly most extent followed by all axes of the northeasterly most extent specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). When using 3D geometries, the elevation of the southwesterly most extent is the minimum elevation in meters and the elevation of the northeasterly most extent is the maximum. |
+| intersects   | GeoJSON Geometry | Searches items by performing intersection between their geometry and provided GeoJSON geometry.  All GeoJSON geometry types must be supported. |
+| ids          | \[string]        | Array of Item ids to return. All other filter parameters that further restrict the number of search results (except `next` and `limit`) are ignored |
+| collections  | \[string]        | Array of Collection IDs to include in the search for items. Only Items in one of the provided Collections will be searched |
+| aggregations | \[string]        | A list of aggregations to compute and return |
+
```

Comment:
This table appears to be a duplicate of the table here? https://github.com/radiantearth/stac-api-spec/tree/master/item-search#query-parameters-and-fields
For maintenance purposes, I wonder if maybe we want to just reference that table here?